src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/debtposition/SynchronizeIngestedDebtPositionActivityImpl.java

```diff
@@ -104,13 +111,17 @@ public SyncIngestedDebtPositionDTO synchronizeIngestedDebtPosition(Long ingestio
         log.info("Synchronization of all debt positions related to ingestion flow file id {} completed", ingestionFlowFileId);
 
         String pdfGeneratedId = null;
+        Path csvPath = null;
         if (!debtPositionsGenerateNotices.isEmpty()) {
             pdfGeneratedId = generateNoticeService.generateNotices(ingestionFlowFileId, debtPositionsGenerateNotices);
+            csvPath = iuvArchivingExportFileService.executeExport(debtPositionsGenerateNotices, ingestionFlowFileId);
         }
 
         return SyncIngestedDebtPositionDTO.builder()
                 .errorsDescription(errors.toString())
                 .pdfGeneratedId(pdfGeneratedId)
+                .iuvFileName(Objects.requireNonNull(csvPath).getFileName().toString())
```

Comment:
here you cannot do requireNonNull... you are explicitly allowing the variable to be null if debtPositionsGenerateNotices is empty... thus csvPath could be null...